src/ragger/backend/interface.py

```diff
@@ -313,3 +313,65 @@ def compare_screen_with_snapshot(self,
         :rtype: bool
         """
         raise NotImplementedError
+
+    @abstractmethod
+    def wait_for_screen_change(self, timeout: float = 10.0, context: list = []):
```

Comment:
Could the type of context be something else than a list? Maybe `Any` or something like that?

Reply:
`Any` would make everything even more opaque... Not very fond of this idea.

Reply:
Actually... If we don't know what the backend returns with `get_screen_content` we should maybe go with what you suggest.

Reply:
Done.